hfo/hfo.py

```diff
@@ -157,8 +169,9 @@ def step(self):
     self.say_msg = ''
     
     # Get the current game status
-    data = self.socket.recv(struct.calcsize("i"))
-    status = struct.unpack("i", data)[0]
+    data = self.socket.recv(struct.calcsize("iii"))
+    status = struct.unpack("iii", data)[0]
```

Comment:
Lets unpack this in one line:
status, side, unum = struct.unpack("iii", data)
self.player_on_ball = HFO_Player(side, unum)